framework/src/modules/http_api/controllers/peers.js

```diff
@@ -68,25 +68,15 @@ PeersController.getPeers = async function(context, next) {
 	filters = _.pickBy(filters, v => !(v === undefined || v === null));
 
 	try {
-		const data = await channel.invoke('chain:getPeers', {
```

Comment:
If we remove this call, we don't need to expose the actions from chain module. `getPeers` & `getPeersCountByFilter` can be removed from:
https://github.com/LiskHQ/lisk/blob/3a9d826dd84b222049b19df4a9eb508ed7624374/framework/src/modules/chain/chain.js#L218
https://github.com/LiskHQ/lisk/blob/3a9d826dd84b222049b19df4a9eb508ed7624374/framework/src/modules/chain/index.js#L61